go-controller/pkg/ovn/egressip_local.go

```diff
@@ -10,6 +10,7 @@ import (
 
 const (
 	// In case we restart we need accept executing ovn-nbctl commands with this error.
+	// The ovn-nbctl API does not support `--may-exist` for `lr-policy-add`
 	policyAlreadyExistsMsg = "Same routing policy already existed"
```

Comment:
This is only used from `egressip.go`, not `egressip_local.go` so it ought to be there?